fe/fe-core/src/main/java/com/starrocks/common/util/RuntimeProfile.java

```diff
@@ -330,9 +330,13 @@ private void update(List<TRuntimeProfileNode> nodes, Reference<Integer> idx, boo
                         // Running profile will report multiple times, we only need the last time value
                         if (tcounter.isSetMin_value()) {
                             counter.setMinValue(tcounter.getMin_value());
+                        } else {
+                            counter.clearMinValue();
```

Comment:
why ? 

if a BE profile lacks MIN/MAX, which probably means there's no skew in it, but the global MIN/MAX should be kept

Reply:
I think merge is done in `mergeIsomorphicProfiles`.  `update` just keeps the sub profile to the newest by `replace`, for example after enabling running profile, BE reports profile for a fragment instance multiple times, and FE just needs to replace the existing profile with the newest. Then merge the profiles of all fragment instance to get the global profile